### PR TITLE
[Do not merge] Fixes #2472 - Makes a first pass at validation issues

### LINKED ISSUE #2472: Validation of webcompat rendered HTML
I was investigating something else and wanted to know if we had issues due to markup.

Steps to reproduce
1. Generate a page
2. View source
3. Copy and paste into the text field of https://html5.validator.nu/

Enjoy!

Example. `/issues/new` route.

capture d ecran 2018-05-31 a 13 58 05

### PR DESCRIPTION
**Pull request by karlcow**
This PR fixes issue #2472

## Proposed PR background

This just intends to fix some markup and validation issues.
I was investigating something else which was not working well. And it was getting into the way.
It seems unrelated, but the HTML code could benefit of a bit of a clean up. 

Anyone can take over this. This is not mine. :)

### CODE REVIEW

**Comment by karlcow** [comment]
@zoepage pinging you on this. Because I probably messed up some of the code/template decision you made. Sorry! If there are better ways to fix I'm all hears.

**Comment by zoepage** [comment]
Thanks for the ping @karlcow. I'm @ a conference and won't have time before Tuesday. (Sorry :/ )


If this is too long, maybe @regaddi could take a look?

File: webcompat/templates/home-page/form.html [hunk 1]
```diff
@@ -73,7 +73,7 @@ <h1 class="headline-1">Report Site Issue</h1>
     </div>
     <div class="form-radio form-element js-Form-group">
       <div class="js-Form-information form-label-message">
-        {{ form.browser_test.label(class_='form-label') }}
+        <div class="form-label">Did you test in another browser?</div>
```
Comment (zoepage) [inline webcompat/templates/home-page/form.html]: I might rather go with label than div here.

File: webcompat/templates/home-page/form.html [hunk 2]
```diff
@@ -26,7 +26,7 @@ <h1 class="headline-1">Report Site Issue</h1>
     </div>
     <div class="form-radio form-element js-Form-group">
       <div class="js-Form-information form-label-message">
-        <label class="form-label form-required" for="problem_category">Type of bug</label>
+        <label class="form-label form-required">Type of bug</label>
```
Comment (zoepage) [inline webcompat/templates/home-page/form.html]: Could you maybe explain, why you removed the for attr? :)

File: webcompat/templates/contributors.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 {% block body %}
 <div>
   {% include "nav/nav.html" %}
-  <main role="main">
```
Comment (patrickhlauke) [inline webcompat/templates/contributors.html]: random note that `<main role="main">` (here and in the other cases) is still necessary for IE11 compat (as it doesn't know, so doesn't expose, the main landmark role)

**Comment by miketaylr** [comment]
Closing per https://github.com/webcompat/webcompat.com/pull/2484#issuecomment-395070952. If there's stuff not covered someone feels strongly about, please file a new issue.

**Pull request closed without merging**